client/src/components/map.module.scss

```diff
@@ -2,4 +2,56 @@
   height: 676px;
   margin-bottom: 29px;
   max-width: revert;
+  margin-top: 50px;
+}
+
+.popupContainer {
+  position: absolute;
+  background-color: white;
+  box-shadow: 0 1px 4px rgba(0, 0, 0, 0.2);
+  padding: 15px;
+  border-radius: 10px;
+  border: 1px solid #cccccc;
+  bottom: 12px;
+  left: -50px;
+  min-width: 280px;
+}
+
+.popupContainer:after,
```

Comment:
Think these can be nested (scoped) in scss?  ... but not critical at all. Also, it gets complicated with "common css"  for `:after` & `:before` then the adding/overriding some these attributes.

e.g.
```scss
.popupContainer {
  :after, :before {
```